src/main/kotlin/com/github/shiraji/ipgwizard/IPGWizardBuilder.kt

```diff
@@ -151,7 +151,8 @@ class IPGWizardBuilder : AbstractExternalModuleBuilder<GradleProjectSettings>(Pr
     }
 
     private fun setupSourceDirectory(modifiableRootModel: ModifiableRootModel, modelContentRootDir: VirtualFile) {
-        val resourceRootPath = "${modelContentRootDir.path}/src/main/java"
+        val languageDirectoryName = if (language == IPGWizardSupportLanguageStep.Language.KOTLIN.name) "kotlin" else "java"
+        val resourceRootPath = "${modelContentRootDir.path}/src/main/${languageDirectoryName}"
```

Comment:
This is really really really small thing (i hate to point these small syntax, but kotlin plugin does.) `${languageDirectoryName}` should be `$languageDirectoryName`